docs/getting-started/locally.md

```diff
@@ -0,0 +1,107 @@
+# nio Local Quickstart
+
+The cloud is an easy way to get the nio Platform up and running, but to see the power of nio as a distributed system, you should install and run nio on a locally or at the edge.
+
+>**[info] Requirements**
+>
+>* **Python3**: https://www.python.org/downloads (you need Python 3.4+)
+>* **nio**: https://app.n.io/binaries/download (you'll need to agree to the license)
+>  * Windows users should download the Windows Installer. Double-click and follow the prompts.
+
+## Installation
+
+Open a terminal (Applications > Utilities > Terminal on MacOS), and install nio using Python’s pip3 installer (verify the filepath and binary filename below):
+```
+pip3 install -U ~/Downloads/nio_lite-20180105-py3-none-any.whl
+```
+> MacOS Users: You may be prompted to install a dependency called "clang". This is a standard component that comes directly from Apple, so it's safe to click "OK" and re-run the command above.
+
+You can test that nio is correctly installed by running the following command:
+```
+which niod
+```
+> If you don’t see a path to niod, make sure Python’s binary directory is on your PATH.
+
+> Help setting your PATH: [Windows](https://msdn.microsoft.com/en-us/library/aa922003.aspx) / [MacOS](http://osxdaily.com/2014/08/14/add-new-path-to-path-command-line/).
+
+## Create a Project
+
+Projects are collections of services, blocks and configurations. Create a nio folder to keep things tidy:
+```
+mkdir -p nio/projects && cd nio/projects
+```
+Use the nio CLI (Command Line Interface) to create a new project named "my_project":
+```
+nio new my_project
+```
+**If you'd rather install the Plant Service from our [Distributed Demostration](https://workshops.n.io/plant/virtual/), enter the following:**
+```
```

Comment:
Do we want this to link them to the `virtual` page or the beginning of the distributed demonstration (the overview of what it is)?

Reply:
good point... changed